### ISSUE TITLE
Stage icon

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit/dashboard#98]
Would it be hard to use the stage icon in the stage deployment?

It's not so important but would be nice to match the behaviour on Github/GitLab where we use different avatars.

![Screenshot from 2021-04-27 07-54-37](https://user-images.githubusercontent.com/20214043/116192246-dabb3180-a72d-11eb-80c6-4b8dd0a52726.png)

![Screenshot from 2021-04-27 07-54-24](https://user-images.githubusercontent.com/20214043/116192249-db53c800-a72d-11eb-9bb2-db697792d598.png)

![Screenshot from 2021-04-27 07-56-02](https://user-images.githubusercontent.com/20214043/116192379-0dfdc080-a72e-11eb-9737-750b98e1b7af.png)

**Comment by IceWreck** [comment]
Would be simpler once #96 is done because its compiling different versions of the dashboard at build time instead of modifying differences between stg and prod on the fly at every request.

**Comment by IceWreck** [comment]
Speaking of #96, I see the image is built here:
https://github.com/packit/dashboard/blob/main/.github/workflows/build-and-push.yml#L41
The tags specified are : `${{ steps.branch_tag.outputs.tag }}`, but is this a list of variables ?

We probably need separate tasks for stg and prod tags because we need to specify separate build tags in both

**Comment by lachmanfrantisek** [comment]
> The tags specified are : `${{ steps.branch_tag.outputs.tag }}`, but is this a list of variables ?

This is only an image tag, not the variables. (`registry/image:tag`)

What we need is probably `build_args` (See the options here: https://github.com/marketplace/actions/buildah-build)


And btw, thanks for working on it!

**Comment by mfocko** [comment]
> Speaking of #96, I see the image is built here:
> https://github.com/packit/dashboard/blob/main/.github/workflows/build-and-push.yml#L41
> The tags specified are : `${{ steps.branch_tag.outputs.tag }}`, but is this a list of variables ?

Those are tags for image, correct me if I'm wrong, `main` gets built as `stg` and `latest`, `stable` gets built as `prod` and then you can use those images as `quay.io/packit/dashboard:stg` for example...

In the related issue I think I linked the docs to the buildah GitHub action... they have a separate field for specifying the `ENV`, looks like `envs`

**Comment by jpopelka** [comment]
I guess this is still nice to have.
The #96 has been merged and the [buildah-build action in the github workflow already uses build-args](https://github.com/packit/dashboard/blob/34668d23c1abd39c52db54f8939d3989df2c8c83/.github/workflows/build-and-push.yml#L51).
Are there any unanswered questions here?

**Comment by IceWreck** [comment]
Sorry, had exams.

The packit logo is used at 3 places. The topbar (defined in JS), the favicon (in the base HTML) and the high res icon that the browser uses for pinned sites (defined in `manifest.json`). 

So I was thinking that instead of adding logic to auto switch between stg and prod icon in all of these places, we can just add another task in `recipe.yaml` to check the environment variable (provided by build args) and if it's staging then rename `logo_stg.png` to `logo.png` 

Is that fine ?
